docker-compose.apps.yml

```
@@ -97,6 +96,36 @@ services:
         condition: service_healthy
+  user-dev-db:
+    image: postgres:17-alpine
+    container_name: user-dev-db
+    ports:
+      - "15432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=user-dev-db
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)